### ISSUE TITLE
What does  self.rgb_lines = list(lines) * 10 mean?

### ISSUE THREAD

**Issue by litingfeng** [laura-wang/video-pace#15]
Hi,

Thank you for your great work.

While I want to apply this method on my custom data, I found this line in ucf101.py.

Why do you multiply 10 here?

Thank you

**Comment by laura-wang** [comment]
Hi,

Thanks for your interests in our work.

Regarding the epoch number, we follow the work "Self-Supervised Learning by Cross-Modal Audio-Video Clustering" to allow more temporal augmentation in the same epoch. And the total training epoch is 180, actually.  In practice, we find it works pretty well so we keep this practice. 

Probably，to just directly use 180 epochs should  also work well (the learning rate schedule should be adjusted accordingly). 
You can try on your own~
